frontend-cert/react-projects/react-props/user-stories.md

```diff
@@ -0,0 +1,22 @@
+# --description--
+
+In this lab, you will create a mood board using reusable React components. The CSS has already been provided for you.
+
+Fulfill the user stories below and get all the tests to pass to complete the lab.
+
+**User Stories:**
+
+1. You should have a `div` with the `id` of `root`.
+2. You should have a `MoodBoardItem` component that uses props and renders a `div` with the `class` of `mood-board-item`.
```

Comment:
we don't need user stories for this because that will already be apart of the boilerplate